VideoWeb/VideoWeb/ClientApp/src/app/home/navigator/navigator.component.ts

```diff
@@ -34,7 +34,12 @@ export class NavigatorComponent implements OnInit {
             this.router.navigate([pageUrls.JudgeHearingList]);
         } else if (userProfile.role === Role.VideoHearingsOfficer) {
             this.router.navigate([pageUrls.AdminVenueList]);
-        } else if (userProfile.role === Role.Representative || userProfile.role === Role.Individual) {
+        } else if (
+            userProfile.role === Role.Representative ||
+            userProfile.role === Role.Individual ||
+            userProfile.role === Role.QuickLinkParticipant ||
+            userProfile.role === Role.QuickLinkObserver
```

Comment:
We use this logic 3 different places and we've seen issues before where not all instances get changed. Might be worth mapping the type we get from the api client to one defined for the UI with a get method to do this for us. I think it's okay to leave that for future improvements though as this was already here.

Reply:
@FarronF  agree to your remark here. We should not carry out any business process check in the UI. UI should be dumb enough not to perform logic and that has to be in the backend and we pass only a boolean field to UI; on that boolean value we perform decision & action.